src/core/hash.js

```diff
@@ -19,15 +19,16 @@ function hashCode(str) {
  * Calculates simple hash value of an array of numbers. Designed for performance, but provides good
  * distribution with small number of collisions.
  *.
- * @param {Uint32Array} typedArray - Array of numbers to hash.
+ * @param {number[]|Uint32Array} typedArray - Array of numbers to hash.
  * @returns {number} Hash value.
+ * @ignore
  */
-function fnv1aHashUint32Array(typedArray) {
+function fnv1aHashUint32Array(array) {
```

Comment:
If the function can now also take a number array, is it now misnamed?

Reply:
I knew you would not miss it .. renaming it.